src/constructor.rs

```diff
@@ -0,0 +1,340 @@
+use std::sync::Arc;
+use teloxide::types::ReplyMarkup;
+use teloxide::{payloads::SendMessageSetters, prelude::Requester, types::Message, Bot};
+use tracing::instrument;
+use crate::database::connection::{CreateQuiz, RetreiveQuiz};
+use crate::database::quiz::{Answer, Question, Quiz};
+use crate::keyboard::{action_keyboard, yes_no_keyboard};
+use crate::state::{QuizData, QuizState};
+use crate::{HandlerResult, UserDialogue};
+
+#[instrument(level = "info", skip(connection))]
```

Comment:
Do you need to logging variables Bot&dialogue all the time? it is a bit excessively

Reply:
Thank you for your review. I removed bot and dialogue records from logger.